pkg/k8s/apis/cilium.io/client/register.go

```diff
@@ -65,6 +65,21 @@ const (
 	// BGPPCRDName is the full name of the BGPP CRD.
 	BGPPCRDName = k8sconstv2alpha1.BGPPKindDefinition + "/" + k8sconstv2alpha1.CustomResourceDefinitionVersion
 
+	// BGPCCCRDName is the full name of the BGP Config CRD.
+	BGPCCCRDName = k8sconstv2alpha1.BGPCCKindDefinition + "/" + k8sconstv2alpha1.CustomResourceDefinitionVersion
+
+	// BGPPCCRDName is the full name of the BGP PeerConfigRef CRD.
+	BGPPCCRDName = k8sconstv2alpha1.BGPPCKindDefinition + "/" + k8sconstv2alpha1.CustomResourceDefinitionVersion
+
+	// BGPACRDName is the full name of the BGP Advertisement CRD.
+	BGPACRDName = k8sconstv2alpha1.BGPAKindDefinition + "/" + k8sconstv2alpha1.CustomResourceDefinitionVersion
+
+	// BGPNCCRDName is the full name of the BGP Agent CRD.
+	BGPNCCRDName = k8sconstv2alpha1.BGPNCKindDefinition + "/" + k8sconstv2alpha1.CustomResourceDefinitionVersion
+
+	// BGPNCOCRDName is the full name of the BGP Node Config Override CRD.
+	BGPNCOCRDName = k8sconstv2alpha1.BGPNCOKindDefinition + "/" + k8sconstv2alpha1.CustomResourceDefinitionVersion
```

Comment:
I know this is sticking to the existing convention, but having to count that (for example) it's _exactly_ three `C`s in `BGPCCCRDName` doesn't sound ideal.

I guess that IDEs will handle this, so this ones non-blocking feedback, especially since I can't think of anything better.

Reply:
I agree, these short names were getting out of hand. I expanded it for better readability. 

Please have a look : 
https://github.com/cilium/cilium/compare/516204e8ac5b8c3d035b14797fc8b773d78f4f7b..06459b56e6e76781e8fc3259f57764d56f464f64